docs/reference/commands/ddl/create_view.md

```diff
@@ -3,3 +3,88 @@ title: CREATE VIEW
 ---
 
 # CREATE VIEW
+
+The `CREATE VIEW` statement creates a named view that represents a stored query. Views can be referenced in queries just like tables, but they don't store data themselves - they execute their underlying query when accessed.
+
+> Note: Currently, only temporary views are supported using `CREATE TEMP VIEW`.
+
+## Syntax
+
+```sql
+CREATE TEMP VIEW [IF NOT EXISTS] view_name [(column_name [, ...])]
+AS query
+```
+
+Where:
+- `TEMP`: Specifies that the view is temporary and will only exist for the current session
+- `IF NOT EXISTS`: Optional clause that prevents an error if a view with the same name already exists
+- `view_name`: Name to assign to the view
+- `column_name`: Optional comma-separated list of names to assign to the view's columns
+- `query`: A SELECT statement that defines the view
```

Comment:
```suggestion
```sql
CREATE TEMP VIEW view_name [(column_name [, ...])]
AS query
```

Where:
- `TEMP`: Specifies that the view is temporary and will only exist for the current session
- `view_name`: Name to assign to the view
- `column_name`: Optional comma-separated list of names to assign to the view's columns
- `query`: A SELECT statement that defines the view
```

IF NOT EXISTS isn't parsing correctly currently.